lib/Time/Duration/Concise.pm

```diff
@@ -207,8 +203,9 @@ sub as_concise_string {
     $precision ||= 10;
     my $time_frames         = $self->_duration_array($precision);
     my @concise_time_frames = map {
-        $_ =~ s/\s+//ig;
```

Comment:
coud you try just: 
```
s/\s+//gr
```

? the `$_ =~` is done explicitely, and `r` flag means return modified value and keep original untouched, rather then  modify original. Aslo, we don't need `i` here (ignore case)